src/layouts/_header.scss

```diff
@@ -0,0 +1,27 @@
+.navigation {
+  float:right;
+  display: block;
+  margin-right: 15px;
+  margin-top: 7px;
+  padding-top: 0px;
+  @include govuk-font-bold-16;
+
+  a {
+    color: $govuk-white;
+    text-decoration: none;
+
+    &:hover {
+     text-decoration: underline;
+    }
+  }
+
+  ul {
+    margin: 0px;
+  }
+
+  li {
+    list-style-type: none;
+    display: inline;
+    margin-left: 15px;
+  }
+}
```

Comment:
missing blank line :trollface:

Reply:
Yeah, honestly need to add it to my editor.